WebApiBackend/WebApiBackendTests/ChoreControllerTest.cs

```diff
@@ -56,17 +58,82 @@ public void Setup()
         }
 
         /// <summary>
-        /// Ensures that a payment can be created for a flat
+        /// Checks for the equality of Chores. This is here because the functionality for checking value equality
+        /// of Chores is unavailable. Because of the rules for Pull Requests, it cannot be added in the same PR as
+        /// the tests and must be added to the Chore class later, if required elsewhere.
+        /// </summary>
+        private bool ChoreEquals(Chore choreOne, Chore choreTwo)
```

Comment:
Just a thought that this could be added as a function to the `Chore` class. Not sure how useful equality checking for chores would be in the actual application itself but if it is useful it might be worth refactoring.

Reply:
This was going to be issue #280 until I convinced myself it was a bad idea because it would overflow into the User class (which is a field of Chore). I wasn't sure if it would be within the scope of a PR for testing.